multi-cluster-spring-boot/.applier/inventory-prod/group_vars/seed-hosts.yml

```diff
@@ -6,6 +6,10 @@ openshift_cluster_content:
     file_action: create
   - name: "create promoter account"
     file: "{{ inventory_dir }}/../projects/promoter-sa.yml"
+  - name: "create prod cluster credential"
```

Comment:
@Gl4di4torRr I'm not completely understanding the addition of this line. There is already an sa created in the line above. Is that not sufficient?